modules/installation-osp-dpdk-binding-vfio-pci.adoc

```diff
@@ -0,0 +1,56 @@
+:_content-type: PROCEDURE
+[id="installation-osp-dpdk-binding-vfio-pci_{context}"]
+= Binding the vfio-pci kernel driver to NICs
+
+Compute machines that connect to a virtual function I/O (VFIO) network require the `vfio-pci` kernel driver to be bound to the ports that are attached to a particularly configured network. Create a machine set for workers that attach to this VFIO network.
+
+.Prerequisites
+
+* The ansible-playbook CLI is installed.
+
+.Procedure
+
+. From a command line, retrieve the VFIO network:
++
+[source,terminal]
+----
+$ openstack network show <VFIO_network_name> -f value -c id
+----
+// TODO: Figure out where this lives.
+
+. Use the `play.yaml` Ansible role: 
```

Comment:
@EmilienM So, the distribution method for [this role](https://github.com/rh-nfv-int/shift-on-stack-vhostuser) isn't okay as-is. To (maybe) get approval, resources that are on GitHub need to be tied to a specific OCP version and must QEed. 

1. Could this be kept elsewhere, e.g. openshift/installer?
2. Is anyone in QE/Support currently familiar with this role?

Reply:
I'll send you another procedure in PM.